core/templates/domain/editor/undo_redo/ChangeObjectFactory.ts

```diff
@@ -26,13 +26,272 @@ import cloneDeep from 'lodash/cloneDeep';
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
+import { MisconceptionBackendDict } from
+  'domain/skill/MisconceptionObjectFactory';
+
+interface CollectionPropertyChange {
+  'cmd': 'edit_collection_property';
+  'property_name': string;
+  'new_value': Object;
```

Comment:
Surely you can be more specific than "Object", here and below?

Reply:
done